src/GovUk.Education.ExploreEducationStatistics.Admin/Migrations/ContentMigrations/20230706081603_MigrateMethodologyInternalReleaseNotes.sql

```diff
@@ -0,0 +1,11 @@
+﻿INSERT INTO [dbo].[MethodologyStatus] (Id, MethodologyVersionId, InternalReleaseNote,
+                                       ApprovalStatus, Created, CreatedById)
+SELECT NEWID() AS Id,
+       MV.Id AS MethodologyVersionId,
+       MV.InternalReleaseNote AS InternalReleaseNote,
+       MV.Status AS ApprovalStatus,
+       MV.Published AS Created,
+       NULL AS CreatedById
+FROM [dbo].[MethodologyVersions] MV
+WHERE MV.Published IS NOT NULL;
```

Comment:
How come they need to be published already?  Do we just want to create ones for status histories that made the Methodology public?

Reply:
My thought was to try and reduce any potential confusion around what MethodologyStatus.Created datetime we set. It seems like a battle between MethodologyVersion.Published and MethodologyVersion.Updated - and I thought the published date is more reliable.

If we migrate `MethodologyVersion.Updated` into `MethodologyStatus.Created` instead, we could use `WHERE MV.Status = 'Approved'` instead.

Maybe that is marginally better, but I'm not fussed about it. Happy to switch to that or something different if you'd prefer.